lib/index.js

```diff
@@ -64,26 +72,26 @@ function authorize(options) {
     };
 
     if(data.xdomain && !data.sessionID)
-      return auth.fail(data, 'Can not read cookies from CORS-Requests. See CORS-Workaround in the readme.', false, accept);
+      return auth.fail(data, ERROR_CORS_REQUEST, false, accept);
 
     auth.store.get(data.sessionID, function(err, session){
       if(err)
-        return auth.fail(data, 'Error in session store:\n' + err.message, true, accept);
+        return auth.fail(data, ERROR_SESSION_STORE + err.message, true, accept);
```

Comment:
Since the session store error is going to be concatenated to this constant, one must search the error for a pattern/regexp that matches this constant, but that is better than that message never being passed at all.